scipy/special/__init__.py

```diff
@@ -95,7 +100,7 @@
    hankel2e      -- Exponentially scaled Hankel function of the second kind.
    wright_bessel -- Wright's generalized Bessel function.
 
-The following is not a universal function:
+The following function does not accept NumPy arrays:
```

Comment:
I wonder if there's a way to come up with a wording which both explains it in simple terms, as this PR does, *and* mentions the word ufunc for those readers who are willing to read up or are familiar with numpy to pick up a term.
(to have a cake and eat it too, yes)

Reply:
What about this:

T"he following function does not accept NumPy arrays (it is not a NumPy ufunc)".

Reply:
This wording works for me and I believe would have worked for me back when I was drowning in various numpy-isms, so a +1 from me. However I'm not necessarily a good proxy and I'm missing the context for this change, so it's my 2 cents.

Reply:
Updated the wording.